src/repo.ts

```diff
@@ -186,17 +184,85 @@ export async function captureChangesInReleaseBranch(
   repositoryDirectoryPath: string,
   { releaseVersion }: { releaseVersion: string },
 ) {
-  await getStdoutFromGitCommandWithin(repositoryDirectoryPath, 'checkout', [
-    '-b',
-    `release/${releaseVersion}`,
-  ]);
   await getStdoutFromGitCommandWithin(repositoryDirectoryPath, 'add', ['-A']);
   await getStdoutFromGitCommandWithin(repositoryDirectoryPath, 'commit', [
     '-m',
     `Release ${releaseVersion}`,
   ]);
 }
 
+/**
+ * This function does create the release branch.
+ *
+ * @param args - The arguments.
+ * @param args.project - Information about the whole project (e.g., names of
+ * packages and where they can found).
+ * @param args.releaseType - The type of release ("ordinary" or "backport"),
+ * which affects how the version is bumped.
+ * @returns A promise for the newReleaseVersion.
+ */
+export async function createReleaseBranch({
+  project,
+  releaseType,
+}: {
+  project: Project;
+  releaseType: ReleaseType;
+}): Promise<{
+  version: string;
+  firstRun: boolean;
+}> {
+  const newReleaseVersion =
+    releaseType === 'backport'
+      ? `${project.releaseVersion.ordinaryNumber}.${
+          project.releaseVersion.backportNumber + 1
+        }.0`
+      : `${project.releaseVersion.ordinaryNumber + 1}.0.0`;
+
+  const releaseBranchName = `release/${newReleaseVersion}`;
+
+  const currentBranchName = await getStdoutFromGitCommandWithin(
+    project.directoryPath,
+    'rev-parse',
+    ['--abbrev-ref', 'HEAD'],
+  );
+
+  if (currentBranchName === releaseBranchName) {
+    debug(`Already on ${releaseBranchName} branch.`);
+    return {
+      version: newReleaseVersion,
+      firstRun: false,
```

Comment:
The purpose of this file was that it would provide some general, single-purpose helper functions for working with a repo. However, the existence of this property makes this more of a higher level function, something that does multiple things rather than one thing. What do you think about moving it to `monorepo-workflow-operations.ts`?